src/prep.ts

```diff
@@ -0,0 +1,25 @@
+import { spawn } from 'child_process'
+import * as fs from 'fs'
+import * as detectIndent from 'detect-indent'
+
+import * as fission from './index'
+import * as util from './util'
+
+export function getEnvVars () : Map <string, string> {
+  let map = new Map()
+  map.set('COMMIT', util.getCommit())
+  map.set('BRANCH', util.getBranch())
+  map.set('PLATFORM', util.getPlatform())
+  return map
+}
+
+export function writeElectronBuilderEnvFile (path : string, envVars : Map <string, string>) {
+  envVars.forEach((val, key) => {
+    // console.log('key', key, 'val', val)
+    fs.appendFileSync(path, `${key}=${val}\n`)
+  })
+}
+
+export function prep (envFilePath : string) {
+  writeElectronBuilderEnvFile(envFilePath, getEnvVars())
```

Comment:
Why the wrapped export? Can't whoever calls this method just call writeElectronBuilderenvFile with correct args?

If so, maybe rename writeElectronBuilderEnvFile to Ping and just have the caller provide args